config/migrations/20200923144227-notule-cleanup1.sparql

```diff
@@ -0,0 +1,23 @@
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX brc: <http://kanselarij.vo.data.gift/id/concept/beslissings-resultaat-codes/>
+PREFIX type: <http://kanselarij.vo.data.gift/id/concept/document-type-codes/>
+
+DELETE {
+  GRAPH ?g {
+       ?notule ext:getekendeDocumentVersiesVoorNotulen ?doc.
+  }
+}
+INSERT {
+  GRAPH ?g {
+       ?zitting dossier:genereert ?doc.
+  }
+}
+WHERE {
+    GRAPH ?g {
+      ?zitting a ?type.
+      ?zitting ext:algemeneNotulen ?notule.
```

Comment:
`?zitting a besluit:Vergaderactiviteit` ?
Vermoedelijk ligt de relatie `ext:algemeneNotulen` enkel tussen Vergaderactiviteit en Notule, maar we willen hier wel zeker van zijn ? 
`?notule a ext:Notule` ook, we willen zeker zijn dat de types overeen komen.